app/post.go

```diff
@@ -1907,6 +1907,23 @@ func (a *App) GetPostsByIds(postIDs []string) ([]*model.Post, int64, *model.AppE
 	return posts, firstInaccessiblePostTime, nil
 }
 
+// GetEditHistoryForPost response bool value indicates, if the post is inaccessible due to cloud plan's limit.
```

Comment:
Is this the correct description for this function?  1/5 this could be removed as I think the function is self-explanatory.

Reply:
You are right, description is not correct. I deleted it as suggested.